cmd/mountlib/mount.go

```diff
@@ -161,6 +163,7 @@ func AddFlags(flagSet *pflag.FlagSet) {
 	flags.BoolVarP(flagSet, &Opt.NetworkMode, "network-mode", "", Opt.NetworkMode, "Mount as remote network drive, instead of fixed disk drive (supported on Windows only)", "Mount")
 	// Unix only
 	flags.DurationVarP(flagSet, &Opt.DaemonWait, "daemon-wait", "", Opt.DaemonWait, "Time to wait for ready mount from daemon (maximum time on Linux, constant sleep time on OSX/BSD) (not supported on Windows)", "Mount")
+	flagSet.BoolVarP(&Opt.Remount, "remount", "", false, "Remount a file system")
```

Comment:
I suggest you wrap this in a `if runtime.GOOS == "linux" { }` then the flag will only appear for linux systems.